modules/ROOT/pages/design-add-api-dependency.adoc

```diff
@@ -49,6 +49,17 @@ include::design-center::partial$design-add-ruleset-dependencies.adoc[]
 
 Even if your projects have not been identified as requiring action, you can proactively check the conformance of your APIs by adding published rulesets as dependencies to your API project. 
 
+When your API fails conformance to rulesets your organization validates against, you get an Email notification. 
+You can click *Code in Design Center* to open API Designer, or click *API Details in Exchange* to be redirected to the API asset Conformance Status page.
+
+.. If the asset version has an editable branch in API Designer, a pop-up appears where you can see the missing ruleset, filtered by tags, category and lifecycle state. To edit those filters, click *Edit criteria*. When you are done, click *Add all rulesets*. +
+_Result_: The rulesets are added to your project.
+
+.. If the asset  version doesn't have an editable branch in API designer, a pop-up appears where you can select an editable branch on your project, or create a new branch from your current version to add the missing rulesets. 
```

Comment:
We would typically say "branch of your project" rather than "branch on your project"

Reply:
There shouldn't be a comma before "create a branch". You may want to pass it by one of the editors, but since this is a long sentence, it would be more clear to say something like "... in API Designer, you can either select an editable branch of your project or create a new branch from the current version. You can then add the missing rulesets to that editable branch."